diff --git a/node_modules/@strapi/admin/admin/src/injectionZones.js b/node_modules/@strapi/admin/admin/src/injectionZones.js
index 533e291..59952ec 100644
--- a/node_modules/@strapi/admin/admin/src/injectionZones.js
+++ b/node_modules/@strapi/admin/admin/src/injectionZones.js
@@ -10,6 +10,9 @@ const injectionZones = {
     tutorials: {
       links: [],
     },
+    homepage: {
+      default: []
+    }
   },
   contentManager: {
     editView: { informations: [], 'right-links': [] },
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index f8b8d85..5b7644e 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -3,23 +3,24 @@
  *
  */
 
+import { Box } from '@strapi/design-system/Box';
+import { Grid, GridItem } from '@strapi/design-system/Grid';
+import { Layout } from '@strapi/design-system/Layout';
+import { Main } from '@strapi/design-system/Main';
+import { LoadingIndicatorPage, useGuidedTour } from '@strapi/helper-plugin';
 import React, { memo, useMemo } from 'react';
-import { FormattedMessage } from 'react-intl';
-import styled from 'styled-components';
 import { Helmet } from 'react-helmet';
+import { FormattedMessage } from 'react-intl';
 import { useHistory } from 'react-router-dom';
-import { LoadingIndicatorPage, useGuidedTour } from '@strapi/helper-plugin';
-import { Layout } from '@strapi/design-system/Layout';
-import { Main } from '@strapi/design-system/Main';
-import { Box } from '@strapi/design-system/Box';
-import { Grid, GridItem } from '@strapi/design-system/Grid';
+import styled from 'styled-components';
 import Logo from '../../assets/images/homepage-logo.png';
-import { useModels } from '../../hooks';
-import isGuidedTourCompleted from '../../components/GuidedTour/utils/isGuidedTourCompleted';
 import GuidedTourHomepage from '../../components/GuidedTour/Homepage';
-import SocialLinks from './SocialLinks';
-import HomeHeader from './HomeHeader';
+import isGuidedTourCompleted from '../../components/GuidedTour/utils/isGuidedTourCompleted';
+import { useModels } from '../../hooks';
+import InjectionZone from '../../shared/components/InjectionZone';
 import ContentBlocks from './ContentBlocks';
+import HomeHeader from './HomeHeader';
+import SocialLinks from './SocialLinks';
 
 const LogoContainer = styled(Box)`
   position: absolute;
@@ -58,6 +59,8 @@ const HomePage = () => {
   }
 
   return (
+    <>
+    <InjectionZone area="admin.homepage.default" />
     <Layout>
       <FormattedMessage id="HomePage.helmet.title" defaultMessage="Homepage">
         {title => <Helmet title={title[0]} />}
@@ -86,6 +89,7 @@ const HomePage = () => {
         </Box>
       </Main>
     </Layout>
+    </>
   );
 };
 
